src/controllers/commands/CommandController.cpp

```diff
@@ -637,6 +638,36 @@ void CommandController::initialize(Settings &, Paths &paths)
             return "";
         });
 
+    this->registerCommand("/popout", [](const QStringList &words,
+                                        ChannelPtr channel) {
+        if (words.size() < 2)
+        {
+            if (!channel->isTwitchChannel() || channel->isEmpty())
+            {
+                channel->addMessage(makeSystemMessage(
+                    "Usage: /popout <channel>. You can also use the "
+                    "command without arguments in any twitch channel to "
+                    "open it's popout chat.."));
+            }
+            else
+            {
+                channel->addMessage(
+                    makeSystemMessage(QString("Opening popout chat for %1...")
+                                          .arg(channel->getName())));
+                QDesktopServices::openUrl(QUrl("https://www.twitch.tv/popout/" +
+                                               channel->getName() +
+                                               "/chat?popout="));
```

Comment:
[`QString().arg()`](https://doc.qt.io/qt-5/qstring.html#arg) syntax seems much cleaner in this case.
I Took care of clang-format
```suggestion
                QDesktopServices::openUrl(
                    QUrl(QString("https://www.twitch.tv/popout/%1/chat?popout=")
                             .arg(channel->getName())));
```